resources/lang/de/filter.json

```
@@ -0,0 +1,6 @@
+{
+  "Select option": "Auswählen",
+  "Press enter to select": "Enter, um auszuwählen",
```

Maybe a better translation:
```suggestion
  "Press enter to select": "Enter drücken, um auszuwählen",
```

---

I deliberately kept it short. There is not much space.